lib/azure/resource_management/ARM_interface.rb

```diff
@@ -385,20 +388,22 @@ def create_virtual_machine(params)
       end
 
       def create_storage_profile(params)
-        Chef::Log.info("Creating Storage Account....")
+        ui.log("Creating Storage Account.... #{("\n")} ")
+
         storage_account = create_storage_account(
           params[:azure_storage_account],
           params[:azure_service_location],
           params[:azure_storage_account_type],
           params[:azure_resource_group_name]
         )
 
-        Chef::Log.info("StorageAccount creation successfull.")
         virtual_hard_disk = get_vhd(
           params[:azure_storage_account],
           params[:azure_os_disk_name]
         )
 
+        Chef::Log.info("StorageAccount creation successfull.")

```

Comment:
`Chef::Log.info` not replaced here with `ui.log` and other thing is why this message is relocated here from its original place.